test/e2e/quick_start_test.go

```diff
@@ -125,6 +125,9 @@ var _ = Describe("When following the Cluster API quick-start with dualstack and
 						ClusterProxy:       proxy.GetWorkloadCluster(ctx, namespace, clusterName),
 						ArtifactsDirectory: artifactFolder,
 						ConfigFilePath:     "./data/kubetest/dualstack.yaml",
```

Comment:
Please also add a TODO in docker.yaml directly above "KUBERNETES_VERSION". This should be hard to miss when we change KUBERNETES_VERSION the next time